bash/atomic

```diff
@@ -651,12 +651,109 @@ _atomic_migrate_import() {
 
 }
 
+_atomic_cluster_no_opts() {
```

Comment:
Never mind, I though this was part of the atomic command not the bash completions.